### PR TITLE
UNIFY-504-test-type-badge-links

### PR DESCRIPTION
**Pull request by jaffrepaul**
https://cypress-io.atlassian.net/browse/UNIFY-504

### CODE REVIEW

File: components/global/E2EOnlyBadge.vue [hunk 1]
```diff
@@ -9,5 +9,7 @@ export default {
 </script>
 
 <template>
```
Comment (jaffrepaul) [inline components/global/E2EOnlyBadge.vue:11]: I originally made this a path prop to pass into the `<Badge />` component but there are edge cases to account for with conditional CSS, etc that felt like a lot since we only need this on these 2 test type badges atm.

**Comment by cowboy** [comment]
The "Component Only" is now in the `unification` branch, so if you rebase, you should be able to add a link to it.

File: components/global/Badge.vue [hunk 2]
```diff
@@ -1,7 +1,9 @@
 <template>
-  <div :class="[$style.badge, $style[type]]">
-    <slot />
-  </div>
+  <router-link :to="{path}" style="border-style: none">
```
Comment (cowboy) [inline components/global/Badge.vue:2]: Do this kind of thing

```vue
<template>
  <router-link v-if="path" :to="{path}">
    <div>some text</div>
  </router-link>
  <div v-else>some text</div>
</template>
```

**Review by cowboy — APPROVED** [review]
